src/pages/import/Data.js

```diff
@@ -59,6 +60,11 @@ export const DataImport = () => {
                             <File name="upload" />
```

Comment:
So I assume this isn't a constant because it isn't reused in the app in other places?

Reply:
Hmm good catch, I'll create an `Upload` component in `src/components/Inputs/`